website/docs/language/upgrade-guides/index.mdx

```diff
@@ -1,138 +1,45 @@
 ---
 page_title: Upgrading to Terraform v1.6
-description: Upgrading to Terraform v1.6
+description: Upgrading to Terraform v1.7
```

Comment:
@liamcervante apologies, I was trying to comment on line 2 as well but it's not in the PR. That one also needs to be updated.

Reply:
Done! Sorry, I didn't read your comment properly and just clicked the "accept suggestion" button 😅